Makefile

```diff
@@ -58,7 +58,7 @@ local_test: generate lint
 local_run: server
```

Comment:
Comment for L41: should we have single place where pipelines are configured? Github Actions can be simulated to run from the laptop as well, that would exercise the same workflow.

Reply:
I've tried to tests Actions locally and failed every time.
It's cumbersome.